src/test/java/org/broadinstitute/hellbender/tools/walkers/vqsr/CNNScoreVariantsIntegrationTest.java

```diff
@@ -40,6 +46,19 @@ public void testAllDefaultArgs() throws IOException {
         spec.executeTest("testInference", this);
     }
 
+    @Test(expectedExceptions = RuntimeException.class)
+    public void testRequirePythonEnvironment() throws IOException {
+        // This test is deliberately left out of the "python" test group in order to ensure that
+        // it only executes when the Python environment has *NOT* been properly established. Also,
+        // skip this test if we're running on the Docker because the Python environment is always
+        // activated there.
+        final String isDockerCI = System.getenv("CI");
+        if (isDockerCI != null && isDockerCI.equalsIgnoreCase("true")) {
```

Comment:
I still maintain that we should change the "CI" variable and set it ourselves as part of the run-docker script so we are not relying on some feature of the environment that might change. Otherwise i'm not too bothered by this being a skip exception.